library-model/test-library-model-generator/src/main/resources/sample_annotated/src/com/uber/nullaway/libmodel/AnnotationExample.java

```diff
@@ -0,0 +1,38 @@
+package com.uber.nullaway.libmodel;
+
+import org.jspecify.annotations.NullMarked;
+import org.jspecify.annotations.Nullable;
+
+@NullMarked
+public class AnnotationExample {
+
+    @Nullable
+    public String makeUpperCase(String inputString) {
+        if (inputString == null || inputString.isEmpty()) {
+            return null;
+        } else {
+            return inputString.toUpperCase();
+        }
```

Comment:
These "stubs" have full implementations of methods. Will that be the case for the JSpecify annotated SDK and other libraries that we want to model as astubx? Or will it be like Checker Framework astub files which are dummied-out to signatures for the most part?

Reply:
This is true for jspecify/jdk (we see full implementations of methods), which is the primary use case we are focused on for the moment.